RELEASE_NOTES.md

```diff
@@ -10,9 +10,81 @@
 * Support for export of SNMP data from multiple switches for population of SNMP Export `grafana` panel
 * Space monitoring improvements - included volumes other than root file system
 
-### Networking
+### DHCP
+
+#### Changed
+
+* **Kea**: Fixed a bug that could cause the auto-repair logic to fail due to an incorrect index
+
+### DNS
+
+#### Changed
+
+* **`ExternalDNS`**: Fixed a bug where `cray-externaldns-manager` could panic if it couldn't connect to PowerDNS on startup
+* **PowerDNS**: Changed `powerdns-manager` SLS error message to debug
+* **PowerDNS**: Fixed bug that caused `powerdns-manager` and externaldns-manager to update the same record
+* **PowerDNS**: Fixed a bug that could cause `powerdns-manager` to intermittently crash when performing a lookup for an existing TSIG key
+* **PowerDNS**: `powerdns-manager` will now retry attempts to add a TSIG key
+* **PowerDNS**: `powerdns-manager` will now create PTR records that are not created by `external-dns`
+* **Unbound**: Changed `cray-dns-unbound` `MaxUnavailable` default to 1 to avoid issues when evicting pods from NCNs
```

Comment:
Might be nice to know the previous value "changed default to 1 from X"

Reply:
```suggestion
* **Unbound**: Changed `cray-dns-unbound` `MaxUnavailable` default from 0 to 1 to avoid issues when evicting pods from NCNs
```